gqlgen.yml

```diff
@@ -14,6 +14,8 @@ model:
 
 # Use []Thing instead of []*Thing
 omit_slice_element_pointers: true
+struct_fields_always_pointers: false
+resolvers_always_return_pointers: false
```

Comment:
Could you add some comments to explain why we do this?

Reply:
added comments, please let me know if I missed something.